cnames_active.js

```diff
@@ -386,6 +386,7 @@ var cnames_active = {
   "cork": "davej.github.io/CorkJS",
   "corki": "dvtate.github.io/corki",
   "corps": "copay.github.io",
+  "cosova": "hardo.github.io",
```

Comment:
```suggestion
  "cosova": "hardo.github.io/cosova",
```

I assume you are wanting to target https://github.com/hardo/cosova?
If so, please update your cnames_active.js entry to be `hardo.github.io/cosova` :)